README.md

```diff
@@ -15,6 +15,11 @@
       alt="GitHub" />
   </a>
 
+  <a href="https://discord.com/invite/7rfW4kg6B5">
```

Comment:
Should we add the Invite URL or the actual server URL, or both? I'd say I'd want to click the URL even if I'm already in the server.

Reply:
@martinlingstuyl when you're already member of the server than the invite URL will redirect you to the server.

Reply:
It shows me this, while I'm already in the server as well... and then it directs me to the rules page. While it does work, it's got all the hallmarks of accepting an invitation.

![image](https://user-images.githubusercontent.com/5267487/194928363-b0040327-8f9f-4abb-804e-19e2542d8b8f.png)

Reply:
Hey @Jwaegebaert, you read this? 😄

Reply:
We can change the link to the `general` channel but users who aren't connected won't be able to use it. So I prefer the invite link to a direct link. Once they are part of the server, I don't think they will need a quick link to `general` from the docs site.

Reply:
We could a guide on how to get started with discord for example. There we could add multiple links to the invite page and some links to direct channels.

Reply:
> Once they are part of the server, I don't think they will need a quick link to `general` from the docs site.

okay, that's a good point. people will probably not click it anymore indeed. Let's keep it as it is in that case.